packages/@sanity/core/src/commands/dataset/listCopyDatasetJobs.js

```diff
@@ -0,0 +1,30 @@
+import lazyRequire from '@sanity/util/lib/lazyRequire'
+
+const helpText = `
+    Options
+        --offset Start position in the list of jobs
+        --limit Maximum number of jobs returned
```

Comment:
Do we have a default or maximum on limit?

Reply:
added.